src/modules/todolist.js

```diff
@@ -0,0 +1,81 @@
+import ToDo from './todo.js';
+
+export default class ToDoList {
+  constructor() {
+    this.toDos = [];
+  }
+
```

Comment:
- [OPTIONAL] Please it will be nice to also implement the solution using an array some simple `Todo tasks` for learning purposes.